source/agora/test/Byzantine.d

```diff
@@ -304,29 +294,48 @@ unittest
     txes.each!(tx => node_1.putTransaction(tx));
     network.setTimeFor(Height(1));  // trigger consensus
 
-    waitForCount(conf.validators - 1, &network.nominate_count, "nominate");
-    waitForCount(conf.validators - 1, &network.prepare_count, "prepare");
-    waitForCount(conf.validators - 1, &network.confirm_count, "confirm");
-    waitForCount(conf.validators - 1, &network.externalize_count, "externalize");
+    waitForCount(conf.validators - 1, &network.envelope_type_counts.nominate_count, "nominate");
+    waitForCount(conf.validators - 1, &network.envelope_type_counts.prepare_count, "prepare");
+    waitForCount(conf.validators - 1, &network.envelope_type_counts.confirm_count, "confirm");
+    waitForCount(conf.validators - 1, &network.envelope_type_counts.externalize_count, "externalize");
 }
 
-/// Too many byzantine nodes will prevent block from being externalized
+/// Half nodes not signing correctly WILL NOT prevent block from being externalized if quorum_threshold = 49
 unittest
 {
-    TestConf conf = { validators : 10, quorum_threshold : 66 };
+    TestConf conf = { validators : 6, quorum_threshold : 49 };
```

Comment:
I think this will actually round up to 50%. Because if `49%` was rounded down then it would imply a quorum slice is 2/6 nodes, but the v-blocking set is then 4 and this would trigger a failure here: https://github.com/bpfkorea/agora/blob/8cd8435497648dd938f99c0803283e36c42b1034/source/scpp/src/scp/QuorumSetUtils.cpp#L95

There is a way to check what is actually being generated here. Call `getQuorumConfig` on the nodes and check it, just like here: https://github.com/bpfkorea/agora/blob/8cd8435497648dd938f99c0803283e36c42b1034/source/agora/test/QuorumShuffle.d#L57

Reply:
Thanks, I have updated the tests to be clearer on how many nodes we are requiring in each test.